helmfile/charts/hnc/templates/user-rbac.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.rbac.create -}}
+{{- range .Values.rbac.user.namespaces }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    {{- include "hnc.labels" $ | nindent 4 }}
+  name: {{ include "hnc.fullname" $ }}-user-role
+  namespace: {{ . }}
+rules:
+{{- if $.Values.rbac.user.allowConfigEdit }}
+- apiGroups:
+  - hnc.x-k8s.io
+  resources:
+  - hierarchyconfigurations
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+{{- end }}
```

Comment:
I suggest that they should always be able to `get,list,watch` the hierachyconfig, maybe even the cluster config `hncconfiguration`.